services/ui-src/src/components/cards/EvaluationPlanEntity.test.tsx

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from "@testing-library/react";
+import { testA11y } from "utils/testing/commonTests";
+import { mockEvaluationPlanFormattedEntityData } from "utils/testing/mockEntities";
+import { EvaluationPlanEntity } from "./EvaluationPlanEntity";
+
+const evaluationPlanEntity = (
+  <EvaluationPlanEntity
+    formattedEntityData={mockEvaluationPlanFormattedEntityData}
+  />
+);
+
+describe("<EvaluationPlanEntity />", () => {
+  beforeEach(() => {
+    render(evaluationPlanEntity);
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test("EvaluationPlanEntity renders correctly", () => {
+    expect(
```

Comment:
There are some other scenarios that would be good to test. I'll note in the component file.